mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -236,7 +236,7 @@ function removeTrack() {
     }
     var track = {};
     track.uri = [currentplaylist[i].uri];
-    mopidy.tracklist.remove(track);
+    mopidy.tracklist.remove({'uri':track.uri});
```

Comment:
I'm presuming it works because javascript object notation is basically just
a dictionary and is seen as such by whatever magic js translation mopidy
does. But the latter won't.
On 5 Mar 2015 05:04, "John Cass" <notifications@github.com> wrote:

> In mopidy_musicbox_webclient/static/js/controls.js
> <https://github.com/woutervanwijk/Mopidy-MusicBox-Webclient/pull/64#discussion_r25839855>
> :
>
> > @@ -236,7 +236,7 @@ function removeTrack() {
> >      }
> >      var track = {};
> >      track.uri = [currentplaylist[i].uri];
> > -    mopidy.tracklist.remove(track);
> > +    mopidy.tracklist.remove({'uri':track.uri});
>
> It's interesting that the old code worked for you. Based on the Mopidy
> API docs
> <https://docs.mopidy.com/en/latest/api/core/#mopidy.core.TracklistController.filter>
> I would expect
>
> mopidy.tracklist.remove(uri = track.uri)
>
> to work as well (which is probably easier to read), but not just passing
> track.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/woutervanwijk/Mopidy-MusicBox-Webclient/pull/64/files#r25839855>
> .
>